### ISSUE TITLE
Postgres 17 compatibility

### ISSUE THREAD

**Issue by n-rodriguez** [pawurb/ruby-pg-extras#25]
Hi there!

It seems that `ruby-pg-extras` is not yet compatible with Postgres 17 :

```sh

ActionView::Template::Error (PG::UndefinedColumn: ERROR:  column \"blk_read_time\" does not exist
LINE 6: interval '1 millisecond' * (blk_read_time + blk_write_time) ...
                                    ^
                                    ):

Causes:
ActiveRecord::StatementInvalid (PG::UndefinedColumn: ERROR:  column \"blk_read_time\" does not exist
LINE 6: interval '1 millisecond' * (blk_read_time + blk_write_time) ...
                                    ^
                                    )
PG::UndefinedColumn (ERROR:  column \"blk_read_time\" does not exist
LINE 6: interval '1 millisecond' * (blk_read_time + blk_write_time) ...
                                    ^
                                    )
    1:   h1 class=\"font-bold text-xl my-5\"Diagnose/h1
    2:   table class=\"w-full font-mono border-collapse border my-5\"
    3:     % RailsPgExtras.diagnose(in_format: :hash).each do |diagnosis| %
    4:       tr class=\"%= diagnosis[:ok] ? \"bg-green-300\" : \"bg-red-300\" %\"
    5:         td class='p-1 border font-bold'%= diagnosis[:check_name] %/td
    6:         td class='p-1 border'%= diagnosis[:message] %/td
```

Thank you!

**Comment by n-rodriguez** [comment]
See: https://www.postgresql.org/docs/current/release-17.html#RELEASE-17-MIGRATION

> Rename I/O block read/write timing statistics columns of [pg_stat_statements](https://www.postgresql.org/docs/current/pgstatstatements.html) (Nazir Bilal Yavuz) [git](https://postgr.es/c/13d00729d)
> This renames `blk_read_time` to `shared_blk_read_time`, and `blk_write_time` to `shared_blk_write_time`.

**Comment by pawurb** [comment]
@n-rodriguez thanks! It's now released as `5.4.1`

**Comment by n-rodriguez** [comment]
@pawurb thank you!
